Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -90,6 +113,29 @@ import BraintreeDataCollector
     ) {
         tokenize(request: request, completion: completion)
     }
+
+    /// Tokenize a PayPal request to be used with the PayPal Checkout or Pay Later flow.
+    ///
+    /// - Note: You can use this as the final step in your order/checkout flow. If you want, you may create a transaction from your
+    /// server when this method completes without any additional user interaction.
+    ///
+    /// On success, you will receive an instance of `BTPayPalAccountNonce`; on failure or user cancelation you will receive an error.
+    /// If the user cancels out of the flow, the error code will be `.canceled`.
+    ///
+    /// - Parameter request: A `BTPayPalCheckoutRequest`
+    /// - Returns: A `BTPayPalAccountNonce` if successful
+    /// - Throws: An `Error` describing the failure
+    public func tokenize(_ request: BTPayPalCheckoutRequest) async throws -> BTPayPalAccountNonce {
```

Comment:
Do we also need to make changes to demo app for this?

Reply:
We can punt on this along with tests ([conversation here](https://github.com/braintree/braintree_ios/pull/975#discussion_r1142276481)) I did test them all manually though and you're more than welcome to pull the branch down locally to test as well!